src/LightningExtension/AwaitTrait.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace Acquia\LightningExtension;
+
+use Behat\Mink\Exception\UnsupportedDriverActionException;
+use Drupal\DrupalExtension\Context\MinkContext;
+
+trait AwaitTrait {
```

Comment:
This is gardening of the tests. It's a lot nicer than having to pull in MinkContext and call iWaitForAjaxToFinish().